schema.graphql

```diff
@@ -22,6 +167,22 @@ type Token @entity {
   symbol: String!
   "Token decimals fetched by contract call"
   decimals: Int!
+  ######################################################################################################################
+  amountHistory: [TokenTradingEvent!]! derivedFrom(field: "token")
+  numberOfTrades: Int! # write a how to do it in the query or add the prop. 
+  totalAmountTradedUsd: BigDecimal
+  totalAmountTradedEth: BigDecimal
+}
+
+######################################################################################################################
+type TokenTradingEvent @entity {
+  "kinda token-timestamp"
```

Comment:
kinda? better give meaningful comments

I understand this is a trade for a token, i don't think this is needed. We can make the `history` prop in the token be mapped directly to a `trade`, right?

Reply:
Sorry for the "kinda" it was suposed to be a comment for discussing the Id not for being the final version.

Reply:
We can, but we need to rethink how we can do it because we can't derive from token0 and token1. Because it wouldn't be deterministic. That's why I suggested this intermediate entity.